src/app/build/build.component.ts

```diff
@@ -1,15 +1,43 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, HostBinding } from '@angular/core';
+import { Router, ActivatedRoute, ParamMap } from '@angular/router';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/switchMap';
+
+import { MomentModule } from 'angular2-moment';
+
+import { BuildService } from '../services/build/build.service';
+// import { JobService } from '../job.service';
```

Comment:
And here we need the job service to get the logs?

Reply:
Nope! Removed.